libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -5128,14 +5100,19 @@ MAV_RESULT GCS_MAVLINK::handle_command_storage_format(const mavlink_command_int_
 MAV_RESULT GCS_MAVLINK::handle_command_int_packet(const mavlink_command_int_t &packet)
```

Comment:
This GCS_MAVLINK::handle_command_int_packet() method is going to need to be changed to include a new argument, "const mavlink_message_t &msg"